docs/docs/upgrade/network-api-migration.md

```diff
@@ -11,13 +11,13 @@ This document contains the high-level structural differences between the 1.4 and
 
 ### Important notes
 
-Marathon now supports applications attached to more than one network. However, additional applies when you specify more than one network:
+Marathon now supports applications attached to more than one network. However, additional rules apply when you specify more than one network:
 
-- If a portMapping has defined a hostPort, the networkNames array must be defined with the name(s) of the networks it's referring to.
+- If a `portMapping` has defined a `hostPort`, the `networkNames` array must be defined with the name(s) of the `networks` it's referring to.
```

Comment:
nit: portMapping isn't a field name. I think I used asterisks in the other network docs for it.